compiler/src/iree/compiler/InputConversion/Common/QuantizedMatmulToMatmul.cpp

```diff
@@ -88,34 +106,38 @@ struct QuantizedMatmulToMatmul
     int indexOfLhsZpTimesRhsZpTimesKSizeInput = 0;
     Type accElTy = accType.getElementType();
     if (!rhsZpIsConstantZero) {
+      SmallVector<bool> colRedIterator(lhsRank, false);
+      colRedIterator.back() = true;
       Value lhsSums = sumReduceDimensionSubset(builder, lhs, accElTy,
-                                               /*is_reduction=*/{false, true});
+                                               /*is_reduction=*/colRedIterator);
       indexOfLhsSumsInput = addInput(lhsSums, mapToRowDim);
       indexOfRhsZpInput = addInput(rhsZp, mapToNone);
     }
     if (!lhsZpIsConstantZero) {
+      SmallVector<bool> rowRedIterator(lhsRank, false);
+      rowRedIterator[int(batch)] = true;
       Value rhsSums = sumReduceDimensionSubset(builder, rhs, accElTy,
-                                               /*is_reduction=*/{true, false});
+                                               /*is_reduction=*/rowRedIterator);
```

Comment:
Given that we already have a variable with meaningful name, we can remove the function argument comment. I.e., we can remove `/*is_reduction=*/`. (I missed it in the first round of review..)